examples/clerk-nextjs/pages/sign-up/[[...index]].tsx

```diff
@@ -0,0 +1,7 @@
+import { SignUp } from '@clerk/nextjs'
```

Comment:
Why isn't this just a page named sign-up.tsx?

Reply:
I just copy/pasted from the `hasura` starter. I'll tidy these up later. Right now I've just made this work to start writing the guide.